flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Utils/arinc429.js

```diff
@@ -0,0 +1,43 @@
+class Arinc429Word {
+    constructor(value) {
+        Arinc429Word.f64View[0] = value;
+
+        this.ssm = Arinc429Word.u32View[0];
+        this.value = Arinc429Word.f32View[1];
+    }
+
+    static empty() {
+        return new Arinc429Word(0);
+    }
+
+    static fromSimVarValue(name) {
+        return new Arinc429Word(SimVar.GetSimVarValue(name, "number"));
+    }
+
+    isNormal() {
+        return this.ssm === Arinc429Word.SignStatusMatrix.NormalOperation;
+    }
```

Comment:
What about more utility methods? It seems likely we will need `isFailureWarning`.

Reply:
Yeah. Good point. I only coded the ones that I needed at this point in time, but it's probably better to provide methods for all of them.